mlflow/sagemaker/__init__.py

```diff
@@ -1503,6 +1639,22 @@ def _find_transform_job(job_name, sage_client):
             return None
 
 
+def _find_model(model_name, sage_client):
```

Comment:
Could you change this to does_model_exist(), and only return True or False to limit the scope? The returned model attributes are not really used anywhere.

Reply:
Done.

Reply:
Sure thing. Done.